tester/src/main/java/com/filestack/android/demo/MainActivity.java

```diff
@@ -69,7 +70,7 @@ public void launch(View view) {
         String mimeFilter = sharedPref.getString("mime_filter", null);
         List<String> mimeTypes = Arrays.asList(mimeFilter.split(","));
 
-        String apiKey = sharedPref.getString("api_key", null);
+        String apiKey = "ACGv8iMoCTXS27m716Xsqz";// sharedPref.getString("api_key", null);
```

Comment:
do we need the explicit key for this fix to work?

Reply:
nah, removed